src/any_llm/providers/groq/utils.py

```
@@ -35,6 +35,16 @@
     )
 
 
+def _groq_timing_details(usage: Any) -> dict[str, int | float]:
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `usage`

(ANN401)